sentinelone/komand_sentinelone/connection/connection.py

```diff
@@ -26,13 +29,23 @@ def connect(self, params):
         self.password = params.get(Input.CREDENTIALS).get('password')
         self.url = params.get(Input.URL)
 
+        index = self.url.find("/", self._get_start_index(self.url))
+        if index >= 0:
+            self.url = self.url[:index]
+
         # Add trailing slash if needed
         if not self.url.endswith("/"):
             self.url = self.url + "/"
 
         token = self.get_auth_token(self.url, self.username, self.password)
         self.logger.info("Token: " + "*************" + str(token[len(token) - 5:len(token)]))
 
+    @staticmethod
+    def _get_start_index(url):
+        if url.startswith("http://") or url.startswith("https://"):
+            return 9
```

Comment:
Should we be returning a different index depending on if it is http/https?

Reply:
In reference to @jschipp-r7 comment we will remove 'http'.